TodoList/TodoList/Models/MyDbContext.cs

```diff
@@ -7,13 +7,36 @@ public class MyDbContext: DbContext
         public MyDbContext(DbContextOptions options) : base(options) { }
 
         public DbSet<User> users { get; set; }
+        public DbSet<Category> categories { get; set; }
+        public DbSet<ToDo> toDos { get; set; }
```

Comment:
```suggestion
        public DbSet<User> Users { get; set; }
        public DbSet<Category> Categories { get; set; }
        public DbSet<ToDo> ToDos { get; set; }
```
xem lại convention nhé